controller/src/test/java/io/pravega/controller/server/bucket/PravegaTablesStoreBucketServiceTest.java

```diff
@@ -303,8 +303,18 @@ public void testSplitBrainScenrio() throws Exception {
 
         //Check buckets are evenly distributed
         assertEventuallyEquals(1, () -> bucketManager1.getBucketServices().size(), 10000);
-        assertEquals(1, bucketManager2.getBucketServices().size());
-        assertEquals(1, watermarkingService.getBucketServices().size());
+
+        Set<Integer> bucketsForBucketManager1 = bucketManager1.getBucketServices().keySet();
+        Set<Integer> bucketsForBucketManager2 = bucketManager2.getBucketServices().keySet();
+        Set<Integer> bucketsForBucketManager3 = watermarkingService.getBucketServices().keySet();
+
+        assertEquals(1, bucketsForBucketManager2.size());
+        assertEquals(1, bucketsForBucketManager3.size());
+
+        //Check all the bucket manager have identical buckets.
+        assertFalse(bucketsForBucketManager1.stream().anyMatch(bucketsForBucketManager2::contains));
```

Comment:
I think that either the comment or the assertion are wrong, because the assertion is checking that the buckets of the different bucket managers are disjoint sets, right? But the comment says that they are "identical". Please, check what are the right assertions to be done here.

Reply:
Thanks for highlighting. Yes comment line is wrong. The purpose of this assert is to verify that all bucket managers is having unique buckets running. Same buckets shouldn't be running on different instances.